app/scripts/modules/kubernetes/src/v2/manifest/ManifestQos.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { get } from 'lodash';
+
+import { HoverablePopover } from '@spinnaker/core';
+import './manifestLabels.less';
+
+export interface IManifestQosProps {
+  manifest?: {
+    status?: {
+      qosClass?: string
+    }
+  };
+}
+
+export class ManifestQos extends React.Component<IManifestQosProps> {
+  private qosClass(): string {
+    return get(this.props, ['manifest', 'status', 'qosClass'], 'Unknown');
+  }
+
+  private qosStyle(): string {
+    const qosClass = this.qosClass();
+    if (qosClass === 'Guaranteed') {
```

Comment:
I don't know if I love the colors, since it feels like we're applying judgments to the different QOS classes...like "watch out! using BestEffort QOS is a bad idea!". Which might be annoying for a user that has a completely justified use case for the QOS class.

Reply:
I view the colors as indicating how likely this pod is to be forceably killed/evicted, not a value judgment of the tier.

Reply:
> I view the colors as indicating how likely this pod is to be forceably killed/evicted, not a value judgment of the tier.

I see the point, but if we use the same `warn` colour elsewhere to indicate "watch out! You should investigate" I'd tend to agree with Dan that a user may feel prompted to action when in fact this is more informational.

Reply:
The QOS classes are arguably a niche configuration property, as they are never directly expressed by the user but instead a property of how resource requests are set. In the case where a user knows that they have selected a QOS tier that involves their pod getting evicted, I don't see a warning color as being surprisng/harmful. In the case where a user (and I'm guessing this is the common non-operator user) sees the QOS tier, it's helpful to highlight a common misconfiguration pattern. After all, BestEffort is the lowest guarantee of stability, but also the default class when no special configuration has been made.

Reply:
Ah, thanks for clarifying! If "BestEffort" is a default that users may justifiable want to take action on, a warning seems like a decent option.

Could/should we include some kind of recommendation in the popup?

Reply:
The docs we link to describe how to set the various classes -- if you think it's helpful we can condense that into the popup.

Reply:
@dorbin ^^?